lib/puppet/parser/functions/with.rb

```diff
@@ -3,9 +3,9 @@
   :type => :rvalue,
   :arity => -1,
   :doc => <<-DOC
-Call a [lambda](https://docs.puppetlabs.com/puppet/latest/reference/lang_lambdas.html)
+Call a [lambda](https://puppet.com/docs/puppet/latest/lang_lambdas.html)
 with the given arguments and return the result. Since a lambda's scope is
-[local](https://docs.puppetlabs.com/puppet/latest/reference/lang_lambdas.html#lambda-scope)
+[local](https://puppet.com/docs/puppet/latest/lang_lambdas.html#lambda-scope)
```

Comment:
Anchor doesn't appear to be on the page?

Reply:
:+1: